force-app/main/default/lwc/cssLibrary/cssLibrary.css

```diff
@@ -0,0 +1,15 @@
+c-contact-tile {
+    border: solid var(--lwc-borderWidthThin, 1px) #ecebea;
+    border-radius: var(--lwc-borderRadiusMedium, 4px);
+    padding: 0 var(--lwc-varSpacingXxxSmall, 2px);
```

Comment:
Let's keep the margin setting from `compositionIteration.css` in the `c-contact-tile` css rule. 

```css
margin: var(--lwc-borderWidthThin, 1px) 0;
```

When I tested this, it made no difference to add it to `compositionBasic` but removing it from `compositionIteration` crunches the `c-contact-tile` components together.

Reply:
It is kept in compositionIteration.css as it showcases how we can add additional CSS specific to the components after importing a CSS library. Moving margin to `c-contact-tile `will not impact `compositionBasic` as it has single instance of `c-contact-tile`. Shall I move this or change the comment in compositionIteration.css to '/* Other style rules specific to compositionIteration component */'.

Reply:
Ok. I miss-read that. Thanks, @satyasekharcvb. Looks good.